### DISCUSSION TITLE
How to increase ttl when using rtsp protocol ?

### DISCUSSION THREAD

**Discussion by ThibautDur** [bluenviron/mediamtx#847] [Questions]
Hello !

I am currently using rtsp simple server to stream a video through a router.
But because the ttl of the packets is set to 1, the packages are discarded by the router.
Is there please a way to increase ttl for udp packets using rtsp protocol on rtsp simple server side (configuration option, flag,...) ?

Thank you for your answers !

**Comment by aler9** [comment]
Hello, at the moment there isn't an option, but i gathered some documentation, and it can be done by using `golang.org/x/net/ipv4`:

```go
ipv4.NewConn(udpConn).SetTTL(128)
```

I'll open a feature request that allows to track the issue.

**Reply by aler9** [reply]
https://github.com/aler9/rtsp-simple-server/issues/873
